### PR TITLE
Switch the Preview text emoji to one that exists on Win10

### PR DESCRIPTION
**Pull request by DHowett**

### CODE REVIEW

**Comment by DHowett** [comment]
By team agreement!

**Review by zadjii-msft — APPROVED** [review]
# 🌯

File: src/cascadia/TerminalSettingsEditor/PreviewConnection.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ static constexpr std::wstring_view PreviewText{
     L"\x1b[36m@@ -1 +1 @@\x1b[m\r\n"
     L"\x1b[31m-    Windows Console\x1b[m\r\n"
     L"\x1b[32m+    Windows Terminal!\x1b[m\r\n"
-    L"C:\\> \x1b[93mWrite-Host \x1b[36m\"\xd83e\xde9f!\"\x1b[1D\x1b[m"
+    L"C:\\> \x1b[93mWrite-Host \x1b[36m\"\xd83c\xdf2f!\"\x1b[1D\x1b[m"
```
Comment (lhecker) [inline src/cascadia/TerminalSettingsEditor/PreviewConnection.cpp:19]:
```suggestion
    L"C:\\> \x1b[93mWrite-Host \x1b[36m\"\U0001F32F!\"\x1b[1D\x1b[m"
```
(I have not tested this.)

**Review by lhecker — APPROVED** [review]
🌯

**Pull request merged**